code/game/objects/structures/hand_cart.dm

```diff
@@ -0,0 +1,60 @@
+/obj/structure/hand_cart
```

Comment:
Need a Destroy() override to drop or remove C. Also a physically_destroyed override possibly.